File: src/main/java/com/googongill/aditory/controller/dto/category/UpdateCategoryRequest.java
package com.googongill.aditory.controller.dto.category;

import com.googongill.aditory.domain.enums.CategoryState;
import jakarta.validation.constraints.NotNull;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UpdateCategoryRequest {
    @NotNull
    private String categoryName;
    private CategoryState categoryState;
    private String asCategoryName;
}

File: src/main/java/com/googongill/aditory/controller/dto/category/request/UpdateCategoryRequest.kt
package com.googongill.aditory.controller.dto.category.request

import com.googongill.aditory.domain.enums.CategoryState
import jakarta.validation.constraints.NotNull

class UpdateCategoryRequest(
    val categoryName: @NotNull String,
    val categoryState: CategoryState,
    val asCategoryName: String
)